universal-login-react/src/ui/commons/WaitingFor.tsx

```diff
@@ -34,11 +34,11 @@ const renderWaitingFor = (action: string, relayerConfig: PublicRelayerConfig, tr
   );
 };
 
-export const WaitingFor = ({action, relayerConfig, transactionHash, children, className}: WaitingForProps) => {
+export const WaitingForTransaction = ({action, relayerConfig, transactionHash, children, className}: WaitingForTransactionProps) => {
```

Comment:
Rename file name to WaitingForTransaction.tsx ?

Reply:
👍

Reply:
I will extract WaitingFor and this component will be in this file, so that is why I didn't rename the file name.

Reply:
@rzadp